stories/assets/js/Sidebar_custom.js

```diff
@@ -1,6 +1,6 @@
 export function SidebarNav() {
-  $('#accordion-menu > li.accordion--active').children('.accordion__panel').slideDown();
-  $('#accordion-menu > li span').click(function () {
+  $('.accordion-navbar > li.accordion--active').children('.accordion__panel').slideDown();
+  $('.accordion-navbar > li span').off('click').on('click',function () {
```

Comment:
@neelesh-tippe why do we need to use `.off()` and `.on()`. Any specific reason to use these?
Also can we increase the chevron up and down click area? Right now the anchor link gets clicked on clicking just few pixels above the chevron.

Reply:
I have used sidebar twice in template, because of that click event was triggering twice. To unbind one trigger I have used Off and on.